src/cycax/cycad/cycad_part.py

```diff
@@ -398,51 +398,36 @@ def export(self) -> dict:
         dict_out["parts"] = list_part
         return dict_out
 
-    def render(self, engine: str = "OpenSCAD", engine_config: dict = None, side: str = None) -> dict:
+    def render(self, engine: str = "Preview3D", engine_config: dict = None) -> dict:
         """This class will render the necessary diagrams when called with the following methods. It is invoked int CycadPart and can be called: CycadPart.render.pyplot(left).
         Args:
             engine: Name of the engine to use.
             engine_config: Configuration passed on to the PartEngine. It is engine specific.
-            side: this will be used for pyplot
         """
 
         part_files = {}
         _eng_lower = engine.lower()
         if _eng_lower == "simple2d":
-            # TODO: Update simple2d to match openscad and freecad part engines.
-            # This method will created a pyplot drawing of the object.
-            if side is None:
-                side = "TOP"
-            plotter = Figure(part_no=self.part_no, side=side)
-            plotter.save_as_figure()
+            if "side" in engine_config:
+                side = engine_config["side"]
+            else:
+                side="TOP"
+            part_engine = Simple2D(name=self.part_no, side=side)
```

Comment:
Lets pass in the whole engine config down to the engine. Then setting a default is the problem of the PartEngine Class.